src/components/App/styles.css

```diff
@@ -0,0 +1,30 @@
+.App {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family: 'Raleway', sans-serif;
+}
+
+
+header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-weight: 500;
+}
+
+header h1 {
+    margin-bottom: 0;
+    text-transform: uppercase;
+    font-size: 2.5em;
+    letter-spacing: 2.5px;
+}
+
+header h4 {
+    font-size: 1.25em;
+    margin-top: 5px;
```

Comment:
For margins and paddings, I prefer to use REMs or EMs instead of pixels.  Pixels are a fixed size. Heres a good article for reference as to why. Read the article but the TLDR is that 1rem = 16px.

https://engageinteractive.co.uk/blog/em-vs-rem-vs-px